tokio/Cargo.toml

```diff
@@ -130,6 +130,9 @@ tempfile = "3.1.0"
 async-stream = "0.3"
 socket2 = "0.4"
 
+[target.'cfg(target_os = "freebsd")'.dev-dependencies]
+mio-aio = { git = "https://github.com/asomers/mio-aio", rev = "2f56696", features = ["tokio"] }
```

Comment:
Once this is available in a released version of Tokio, I'd like this to be updated to point at a released version of `mio-aio`.

Reply:
Of course.  I'll release a new version of mio-aio as soon as this PR merges.  I've been holding back just in case Tokio required some changes to mio-aio.